include/BSMPT/bounce_solution/bounce_solution.h

```diff
@@ -81,6 +102,16 @@ class BounceSolution
    */
   bool completion_temp_set = false;
 
+  /**
+   * @brief Temperature at which to calculate parameters
+   * - = 0 - Not set
+   * - = 1 - Approximate Nucleation
+   * - = 2 - Exact Nucleation
+   * - = 3 - Percolation
+   * - = 4 - Completion Temperature
+   */
+  int which_transition_temp = false;
```

Comment:
```suggestion
  int which_transition_temp = 0;
```

I find this highly irritating. Why write false here and convert it to 0?
Also why not use an enum class with these values?

Reply:
I made this an enum class